src/main/java/echo/main/Echo.java

```diff
@@ -0,0 +1,233 @@
+package echo.main;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import echo.exception.EchoException;
+import echo.parser.Parser;
+import echo.storage.Storage;
+import echo.task.Deadlines;
+import echo.task.Events;
+import echo.task.Task;
+import echo.task.ToDos;
+import echo.tasklist.TaskList;
+import echo.ui.Ui;
+
+/**
+ * Represents the Chat Bot called Echo
+ *
+ * @author Ernest Lim
+ */
+public class Echo {
+    private Storage storage;
+    private Ui ui;
+    private TaskList taskList;
+
+    private Parser parser;
+
+    /**
+     * Constructor to create Echo Object
+     *
+     * @param filePath String that contains the file path to store or load the list of task
+     */
+    public Echo(String filePath) {
+        this.ui = new Ui();
+        this.taskList = new TaskList();
+        this.parser = new Parser();
+        this.storage = new Storage(filePath);
```

Comment:
Optional, but you can consider adhering to the advanced [coding standards](https://se-education.org/guides/conventions/java/index.html#variables) set out by this course, which is
- Avoid unnecessary use of this with fields. Use the `this` keyword only when a field is shadowed by a method or constructor parameter.

Since `ui`, `taskList`, `parser` and `storage` are not shadowed by any other parameters here, you can omit the `this` keyword (this would be applicable for every other class as well).